YangLu/task01/quicksort/src/main/kotlin/ru/bmstu/iu4/quicksort_function.kt

```diff
@@ -1,9 +1,8 @@
 package ru.bmstu.iu4
 
-fun sort(sortArray: IntArray, leftIndex: Int, rightIndex: Int) {
-    var leftIndexTemp = leftIndex
-    var rightIndexTemp = rightIndex
-    if (leftIndex > rightIndex)
+fun sort(sortArray: IntArray, leftIndexTemp: Int, rightIndexTemp: Int) {

```

Comment:
What was changed?

Reply:
Sorry I did changed them and pull them on the Github.But I don't why there are old codes here. I'll try to pull again.